tools/integration_tests/write_large_files/concurrent_write_files_test.go

```diff
@@ -0,0 +1,101 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package write_large_files
+
+import (
+	"fmt"
+	"log"
+	"os"
+	"path"
+	"sync"
+	"syscall"
+	"testing"
+
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/operations"
+	"github.com/googlecloudplatform/gcsfuse/tools/integration_tests/util/setup"
+)
+
+const (
+	FileOne               = "fileOne.txt"
+	FileTwo               = "fileTwo.txt"
+	FileThree             = "fileThree.txt"
+	DirForConcurrentWrite = "dirForConcurrentWrite"
+)
+
+func writeFile(fileName string, fileSize int64) (err error) {
+	filePath := path.Join(setup.MntDir(), DirForConcurrentWrite, fileName)
+	f, err := os.OpenFile(filePath, os.O_WRONLY|os.O_CREATE|syscall.O_DIRECT, WritePermission_0200)
+	if err != nil {
+		err = fmt.Errorf("Open file for write at start: %v", err)
```

Comment:
Combine `err = something` and `return err` into `return something` in all the changes in this file.